src/web/mjs/engine/BookmarkManager.mjs

```diff
@@ -13,6 +13,8 @@ export default class BookmarkManager extends EventTarget {
         super();
         this.bookmarks = [];
         this._bookmarkImporter = bookmarkImporter;
+
+        document.addEventListener( EventListener.onSettingsSaved, this.saveProfile.bind( this ) );
```

Comment:
Created new listener onSettingsSaved instead of onSettingsChanged, read comment below

Reply:
Unfortunately i hadn't time yet to refactor `Settings.mjs` to the new event target system, so it seems you need to use the deprecated document based system to use your new event.
In that case you also need to declare a string for this event in [Enums.mjs](https://github.com/manga-download/hakuneko/blob/9a52af9cf2605bdb922a61b323f3b06ddb968220/src/web/mjs/engine/Enums.mjs#L8), where you have do define a string for `EventListener.onSettingsSaved`, otherwise this is `undefined` which may lead to problems.